tests/testthat/test-oc_check_status.R

```diff
@@ -0,0 +1,29 @@
+library("opencage")
+context("oc_check_status")
+# vcrify
```

Comment:
is this a todo item?

Reply:
Yes.

Reply:
Maybe open an issue 😉

Reply:
No because it is not a real TODO, lintr cannot see it is a to do 😁

Reply:
You already [did](#48)! 😉 This was meant as a reminder as to which functions should be tested with vcr, but yeah, lintr will complain, right? 😩